src/Faithlife.Build/BuildApp.cs

```diff
@@ -66,16 +66,31 @@ public BuildTarget Target(string name)
 		return target;
 	}
 
+	/// <summary>
+	/// Adds an action to execute when the command line arguments have been parsed, but before the target(s) are evaluated and executed.
+	/// </summary>
+	/// <param name="commandLineParsed">The action to execute when the command line arguments have been parsed.</param>
+	public void CommandLineParsed(Action<BuildApp> commandLineParsed)
```

Comment:
```suggestion
	public void CommandLineParsed(Action action)
```
Can we use `Action` instead of `Action<BuildApp>`?